src/Service.Host/client/src/components/__tests__/salesPackageTests/salesPackages.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { createShallow } from '@material-ui/core/test-utils';
+import SalesPackages from '../../salesPackages/SalesPackages';
+
+describe('<SalesPackages />', () => {
+    let wrapper;
+    let props;
+    const getTable = () => wrapper.find('WithStyles(Table)');
+    const getRows = () => wrapper.find('WithStyles(TableRow)');
+    const shallow = createShallow({ dive: false });
+
+    describe('when sales packages have loaded', () => {
```

Comment:
Could have tests for when no sales packages and when loading etc.

Reply:
👍